python_ta/__init__.py

```diff
@@ -166,7 +166,7 @@ def reset_linter(config=None, file_linted=None):
         - If the config argument is a dictionary, apply those options afterward.
     Do not re-use a linter object. Returns a new linter.
     """
-    # Tuple of custom options. See 'type' in pylint/config.py `VALIDATORS` dict.
+    # Tuple of custom options. See 'type' in pylint/config/option.py `VALIDATORS` dict.
```

Comment:
This is a great comment to update, and good research done. However, there's no `'type'` in this dict, so it would be more accurate to say that the `'type'` value must be a key in this dictionary.